metainfo.patch

```diff
@@ -0,0 +1,33 @@
+--- a/src/Doom.metainfo.xml.in
++++ b/src/Doom.metainfo.xml.in
+@@ -25,6 +25,15 @@
+       network stack, allowing it to function on modern LANs and the
+       Internet.
+     </p>
++    <p>
++      The easiest way to run @PACKAGE_SHORTNAME@ Doom is to copy a
++      supported game file (known as a "WAD") to the root of your home
++      directory and launch the application.  If you have trouble with
++      the automatic game file detection or need to manage multiple WAD
++      files, you can run the application from the command line with a
++      path to the selected game file:
++      <code>flatpak run io.github.fabiangreffrath.Doom -iwad ~/PATH/TO/GAME.WAD</code>
```

Comment:
Looks like appstream-glib just removes `<code>` with content. Let's remove the tag and keep the command as "plain text".

Reply:
Oh interesting, where are you seeing that?

I see the `<code>` tag preserved in the installed `~/.local/share/flatpak/app/io.github.fabiangreffrath.Doom/current/active/files/share/appdata/io.github.fabiangreffrath.Doom.appdata.xml` file:

```xml
    <p>
      The easiest way to run Crispy Doom is to copy a
      supported game file (known as a "WAD") to the root of your home
      directory and launch the application.  If you have trouble with
      the automatic game file detection or need to manage multiple WAD
      files, you can run the application from the command line with a
      path to the selected game file:
      <code>flatpak run io.github.fabiangreffrath.Doom -iwad ~/PATH/TO/GAME.WAD</code>
    </p>
```

The Flathub front-end seems to preserve the description verbatim: https://github.com/flathub/linux-store-frontend/blob/183d2de5ee8e8656bf9c7dfe35c0dba356361661/src/app/shared/app-details-description/app-details-description.component.html#L7

The back-end seems to as well, although it's harder to trace the logic:
- https://github.com/flathub/linux-store-backend/blob/511af085d6cf292b3f0c7678df7339a715f1ac69/appstream-extractor/appstream-extractor.sh#L75
- https://github.com/flathub/linux-store-backend/blob/511af085d6cf292b3f0c7678df7339a715f1ac69/src/main/java/org/flathub/api/controller/ApiController.java#L33-L36
- https://github.com/flathub/linux-store-backend/blob/fa54b23032bab3a952f9e7163e6d8b360e8a426a/src/main/java/org/flathub/api/dto/AppFullDto.java#L65-L71

Reply:
Actual file being used by backend and native frontends like GNOME Software is generated with appstream-compose. You can view it by running `flatpak run --command=zcat /app/share/app-info/xmls/io.github.fabiangreffrath.Doom.xml.gz`.

Reply:
Ah thank you, I see it now.  I have pushed a change to move the command line into a separate `<p>` tag.